networkx/classes/graphviews.py

```diff
@@ -39,6 +39,50 @@
 
 
 def generic_graph_view(G, create_using=None):
+    """Returns a read-only view of `G`.
+
+    The graph `G` and its attributes are not copied but viewed through the new graph object
+    of the same class as `G` (or of the class specified in `create_using`).
+
+    Parameters
+    ----------
+    G : networkx.Graph
```

Comment:
```suggestion
    G : graph
```
The notation `networkx.Graph` makes it seem that this must be a particular class. But it can be a Graph, DiGraph, MultiGraph or MultiDiGraph, or a subclass of these. We typically use the phrase `graph` for this, or when another graph library version might be relevant the phrase `networkx graph`

Reply:
thanks for the explanation!